bugsnag/src/main/java/com/bugsnag/Configuration.java

```diff
@@ -46,13 +47,12 @@ public class Configuration {
     public boolean sendThreads = false;
```

Comment:
I meant to move it to the above list so that it's declared in the same way as the other config options.

```suggestion
    public boolean sendThreads = false;
    public Serializer serializer = new DefaultSerializer();
```